arch/x86/x86_features.c

```diff
@@ -57,6 +59,16 @@ static void cpuidex(int info, int subinfo, unsigned* eax, unsigned* ebx, unsigne
 #endif
 }
 
+static uint64_t xgetbv(unsigned int xcr) {
```

Comment:
Any function that wraps intrinsic functions should be `static inline`, as Visual C++ doesn't inline the function correctly if it is declared just `static` or `inline`.

Reply:
Should `cpuid` and `cpuidex` be made `static inline` as well?

Reply:
I think so... Those functions most likely were added before I noticed all versions of Visual C++ don't automatically inline short functions.

Reply:
OK, done.